app/views/repositories/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<% provide(:head_title, t("repositories.index.head_title")) %>
+
+<% if current_team %>
+  <%= render partial: "repositories/breadcrumbs.html.erb", locals: { teams: @teams, current_team: current_team, type: @type } %>
+
+  <!-- Nav tabs -->
+  <% if @repositories.present? %>
+    <ul class="nav nav-tabs nav-settings" role="tablist" id="tabs">
+      <% @repositories.each.with_index do |repo, i| %>
+        <li role="presentation" class="<%= 'active' if i == 0 %>">
+          <a class="repository-nav-tab"
+             href="#<%= repo.name.downcase %>"
```

Comment:
Prepend `href` with "custom_repo_", just so there won't be a possibility to interfere with some other ID.